pages/about.html

```diff
@@ -0,0 +1,108 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,400&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="../css/styles.css" />
+    <link rel="stylesheet" href="../css/about.css" />
+  </head>
+  <body>
+    <header class="header">
+      <a href="#">
+        <img src="../imgs/logo-header.png" alt="YTech logo" class="logo" />
+      </a>
+      <button class="btn-mobile-nav icon-open" type="button">
+        <i class="fa-solid fa-bars fa-3x icon-mobile-nav"></i>
+      </button>
+
+      <nav class="main-nav">
+        <ul class="main-nav-list">
+          <li>
+            <button class="btn-mobile-nav icon-close" type="button">
+              <i class="fa-solid fa-xmark fa-4x icon-mobile-nav"></i>
+            </button>
+          </li>
+          <li><a href="../index.html" class="main-nav-link">Home</a></li>
+          <li><a href="#" class="main-nav-link">Careers</a></li>
+          <li><a href="#" class="main-nav-link">Services</a></li>
+          <li><a href="#" class="main-nav-link nav-cta">Contact us</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="section-hero">
+        <div class="hero container">
+          <h3 class="hero-subtitle">&quot;Hello&excl; Sharing World&quot;</h3>
+          <h1 class="hero-title">LET US PROPEL YOUR BUSINESS</h1>
+          <p class="hero-desc">
+            Y-Tech is a Startup private limited company incorporated with the aim to provide
+            information technology solutions that enhance productivity, efficiency and general
+            performance of businesses in Ethiopia. We focus on solutions that improve our customers’
+            business performance by offering fast, reliable and cost effective technological
+            solutions.
+          </p>
+        </div>
+      </section>
+
+      <section class="section-trademark">
+        <h2 class="heading-secondary">Trademark</h2>
+        <div class="partition"></div>
+        <p class="trademark-desc">
+          This name and logo are registered trademarks of Y-Tech Ethiiopia
+        </p>
+        <div class="trademark-img">
+          <img src="../imgs/logo-footer.png" alt="Y-Tech logo" />
+        </div>
+      </section>
+
+      <section class="section-clients">
+        <div class="container">
+          <h2 class="heading-secondary">Clients</h2>
+          <div class="partition"></div>
+          <ul class="clients"></ul>
+        </div>
+      </section>
+    </main>
+
```

Comment:
- [ ] On our about page, we are missing the section displayed below. Since the review guideline expects it, I request we work on it, please.

  ### Desktop
  
  ![image](https://user-images.githubusercontent.com/45002654/175032193-2d40abf7-4d77-445d-81af-4fb07c2e8ed3.png)
  
  ### Mobile
  
  ![image](https://user-images.githubusercontent.com/45002654/175032268-e94bbc20-46a3-4cc5-abcb-086c4df5ce73.png)
  
  This section should be just above the footer.

Reply:
Noted. I will work on it.